<launch>
	<node name="usb_camera_node" pkg="usb_camera" type="usb_camera_node" output="screen" >
		<param name="video_device" type="int" value="0" />
    		<param name="image_width"  type="int" value="1280" />
    		<param name="image_height" type="int" value="720" />
		<param name="frame_id"     type="string" value="titan_camera" />
	</node>
</launch>
